core-kotlin-modules/core-kotlin-concurrency-3/src/main/java/com/baeldung/comparecollectandcollectlatest/UsingCollect.kt

```diff
@@ -0,0 +1,12 @@
+package com.baeldung.comparecollectandcollectlatest
+
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.runBlocking
+
+fun main() {
+    runBlocking {
+        flowOf(1, 2, 3).collect { value ->
+                    println(value)
+        }
+    }
+}
```

Comment:
Should this `println()` be as indented as it is, or should it be less?

```suggestion
fun main() {
    runBlocking {
        flowOf(1, 2, 3).collect { value ->
            println(value)
        }
    }
}
```